web/src/components/tasks/business-formation/validatedFieldsForUser.test.ts

```diff
@@ -47,163 +47,276 @@ describe("validatedFieldsForUser", () => {
+        ...validatedFieldsForAllLegalStructures,
+        "isVeteranNonprofit",
+        "hasNonprofitBoardMembers",
+        "members",
+        "incorporators",
+      ];
+
+      const validatedFieldsArray = validatedFieldsForUser(formationFormData);
+      expect(validatedFieldsArray).toEqual(expect.arrayContaining(expected));
+      expect(validatedFieldsArray).not.toContain("nonprofitBoardMemberQualificationsSpecified");
+      expect(validatedFieldsArray).not.toContain("nonprofitBoardMemberRightsSpecified");
+      expect(validatedFieldsArray).not.toContain("nonprofitTrusteesMethodSpecified");
+      expect(validatedFieldsArray).not.toContain("nonprofitAssetDistributionSpecified");
+    });
+
+    it("isVeteranNonprofit and hasNonprofitBoardMembers radio questions are no", () => {
+      const formationFormData = generateFormationFormData(
+        { hasNonprofitBoardMembers: false },
+        { legalStructureId }
+      );
+
+      const expected: FormationFields[] = [
+        ...validatedFieldsForAllLegalStructures,
+        "isVeteranNonprofit",
+        "hasNonprofitBoardMembers",
+        "members",
+        "incorporators",
+      ];
+
+      const validatedFieldsArray = validatedFieldsForUser(formationFormData);
+      expect(validatedFieldsArray).toEqual(expect.arrayContaining(expected));
+      expect(validatedFieldsArray).not.toContain("nonprofitBoardMemberQualificationsSpecified");
+      expect(validatedFieldsArray).not.toContain("nonprofitBoardMemberRightsSpecified");
+      expect(validatedFieldsArray).not.toContain("nonprofitTrusteesMethodSpecified");
+      expect(validatedFieldsArray).not.toContain("nonprofitAssetDistributionSpecified");
+    });
+
+    it("isVeteranNonprofit and hasNonprofitBoardMembers radio questions are yes", () => {
+      const formationFormData = generateFormationFormData(
+        { hasNonprofitBoardMembers: true },
+        { legalStructureId }
+      );
+
+      const expected: FormationFields[] = [
+        ...validatedFieldsForAllLegalStructures,
+        "isVeteranNonprofit",
+        "hasNonprofitBoardMembers",
+        "members",
+        "incorporators",
+        "nonprofitBoardMemberQualificationsSpecified",
+        "nonprofitBoardMemberRightsSpecified",
+        "nonprofitTrusteesMethodSpecified",
+        "nonprofitAssetDistributionSpecified",
+      ];
+
+      const validatedFieldsArray = validatedFieldsForUser(formationFormData);
+      expect(validatedFieldsArray).toEqual(expect.arrayContaining(expected));
+      expect(validatedFieldsArray).toContain("nonprofitBoardMemberQualificationsSpecified");
+      expect(validatedFieldsArray).toContain("nonprofitBoardMemberQualificationsSpecified");
+      expect(validatedFieldsArray).toContain("nonprofitBoardMemberRightsSpecified");
+      expect(validatedFieldsArray).toContain("nonprofitTrusteesMethodSpecified");
+      expect(validatedFieldsArray).toContain("nonprofitAssetDistributionSpecified");
```

Comment:
[pebble] `nonprofitAssetDistributionSpecified` and all the others are in the `expected` array, so it seems like all these additional expect statements are redundant? Plus, lines 277 and 278 are also the same

Reply:
done